src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,70 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  selectedUserId: number;
+  onClear: () => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC <Props> = ({ selectedUserId, onClear }) => {
+  const userId = selectedUserId;
+  const urlUser = `https://mate.academy/students-api/users/${userId}`;
+
+  const [user, setUser] = useState(
+    {
+      id: 0, name: '', email: '', phone: '',
+    },
+  );
```

Comment:
```suggestion
  const [user, setUser] = useState(null);
```
user can be null, add type for user